src/main/webapp/script.js

```diff
@@ -53,3 +54,14 @@ function displayCalendar(calId) {
   const url = "https://calendar.google.com/calendar/embed?src=" + calId;
   document.getElementById("calendar").src = url;
 }
+
+// delete all secondary calendars created, except for the chosen one 
+function deleteUnchosenCalendars() {
```

Comment:
Slightly nicer/more legible way to write this:

```
function deleteUnchosenCalendars() {
  idArray
    .filter((id, index) => index !== chosenCalendarIndex)
    .forEach(id => deleteCalendar(id));

  const chosenCalendar = idArray[chosenCalendarIndex];
  idArray = [chosenCalendar];

  displayCalendar(chosenCalendar);
}

function deleteCalendar(int calendarId) {
  fetch("/deleteCalendar?calId=" + id, { method: 'POST' });
}
```

Reply:
This is nicer, thanks for showing me! I'll change my code.